lib/Utils/Systemd.pm

```diff
@@ -53,5 +53,23 @@ sub disable_and_stop_service {
     systemctl("stop $service_name", ignore_failure => $args{ignore_failure});
 }
 
+=head2 systemctl
+
+Wrapper around systemctl call to be able to add some useful options.
+
+Please note that return code of this function is handle by 'script_run' or
+'assert_script_run' function, and as such, can be different.
+=cut
+sub systemctl {
+    my ($command, %args) = @_;
+    die "systemctl(): no command specified" if ($command =~ /^ *$/);
```

Comment:
Now that you're touching this part of the code:

use croak instead, and update the mesage: "Invalid call to systemctl: command must be specified"